### DISCUSSION TITLE
tcolorbox 跨页显示的问题

### DISCUSSION THREAD

**Discussion by linsuqinzhe** [tuna/thuthesis#809] [Q&A]
请教，我想在thuthesis模板里用tclorbox宏包。在thusetup里加入了以下代码：

\usepackage[skins,listings,theorems]{tcolorbox}
\tcbuselibrary{skins, breakable, theorems}

一般情况下，编译后都能正常显示，遇到跨页的时候就显示不正常了（分页之后，前面的第1页丢失了）。
我在其他模板里试了一下，跨页却能正常显示。
请问这种情况该怎么解决？谢谢！
![1](https://user-images.githubusercontent.com/33803774/189465647-8a34a9c4-aa93-4be8-bb51-450129f877d6.JPG)
![2](https://user-images.githubusercontent.com/33803774/189465650-2eb73c84-0fd2-45be-b599-de7da176e4db.JPG)

**Comment by xkwxdyy** [comment]
请提供 MWE（最小工作示例）

**Reply by linsuqinzhe** [reply]
在chap01.tex里加入了如下内容：

\begin{tcolorbox}[breakable,colback=gray!10,colframe=black!75!white,title=z]
  1.问题的提出：要清晰地阐述所要研究的问题“是什么”。
  \tcblower
  2.文献综述：对本研究主题范围内的文献进行详尽的综合述评，“述”的同时一定要有“评”，指出现有研究状态，仍存在哪些尚待解决的问题，讲出自己的研究有哪些探索性内容。
\end{tcolorbox}

**Reply by xkwxdyy** [reply]
应该这样给最小工作示例：
```latex
\documentclass{thuthesis}
\usepackage{tcolorbox}
\tcbuselibrary{breakable}

\usepackage{zhlipsum}


\begin{document}

\begin{tcolorbox}[breakable]
  1.问题的提出：要清晰地阐述所要研究的问题“是什么”。
  \tcblower
  2.文献综述：对本研究主题范围内的文献进行详尽的综合述评，“述”的同时一定要有“评”，指出现有研究状态，仍存在哪些尚待解决的问题，讲出自己的研究有哪些探索性内容。
  \zhlipsum[1-4]
\end{tcolorbox}

\end{document}
```

**Reply by linsuqinzhe** [reply]
谢谢您的回答，我的问题还是没有解决。/(ㄒoㄒ)/~~

**Reply by xkwxdyy** [reply]
我并没有回答啥hhh，我只是想说，提供最小示例可以让开发者更好地去解决问题。

**Reply by linsuqinzhe** [reply]
好的，谢谢您。
